app/helpers/ems_infra_helper/textual_summary.rb

```diff
@@ -65,6 +65,7 @@ def textual_guid
   end
 
   def textual_infrastructure_folders
+    return nil if @record.kind_of?(EmsOpenstackInfra)
```

Comment:
I would much prefer to ask the record if it supports some feature/operation and update UI based on that, instead of adding that knowledge to the helper.  This would make the code more resilient to future changes.